lib/mix/lib/mix/tasks/test.ex

```diff
@@ -162,6 +232,14 @@ defmodule Mix.Tasks.Test do
   It must return either `nil` or an anonymous function of zero arity that will
   be run after the test suite is done.
 
+  ### Default coverage module options
+
+    * `:threshold` - summary output colours resulting percentage red/green
```

Comment:
The `:threshold` option seems to only make sense when the `:summary` option is enabled. In order to reflect that hierarchy, what do you think about?

```elixir
test_coverage: [summary: true] # Enables the summary with the default threshold
test_coverage: [summary: [threshold: 70]] # Enables the summary and sets the threshold to 70
```

If in the future the summary adds more configuration variables, they won't be mixed with the global coverage options.